gateway/conf.d/apicast.conf

```diff
@@ -35,6 +35,7 @@ location = /___http_call {
   proxy_set_header User-Agent $user_agent;
   proxy_set_header X-3scale-OAuth2-Grant-Type $grant_type;
   proxy_set_header 3scale-options $options;
+  proxy_set_header uber-trace-id $http_uber_trace_id;
```

Comment:
@mikz @davidor we cannot template this apicast file isn't it? 

This header in jaeger is user customizable, so, we should allow some way to change this via ENV, any idea? some include?